### PR TITLE
fix(FX-4716): update empty-state to match designs

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Fix**

This PR solves [FX-4716]

### Demo
| Before | After |
| ------------- | ------------- |
| before | after |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FX-4716]: https://artsyproduct.atlassian.net/browse/FX-4716?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx [hunk 1]
```diff
@@ -18,24 +18,26 @@ export const ArtworkListEmptyState: FC<ArtworkListEmptyStateProps> = ({
...
 
-      <Spacer x={2} />
+      <Spacer x={[0, 2]} y={[2, 0]} />
 
       <Button
         // @ts-ignore
         as={RouterLink}
-        variant="secondaryBlack"
+        width={["100%", "auto"]}
+        variant="primaryBlack"
```
Comment (anandaroop) [inline src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx:40]: Just wanted to confirm that this is the correct variant, compared to the one in [the Figma screen cap on the card](https://www.notion.so/artsy/UX-mWeb-Empty-state-is-different-from-Figma-design-c1dcb05813fc45d9aec709b57c2fa68c)? 

(Screen cap looks like `primaryGray`? But if not, feel free to merge.)

**Review by anandaroop — APPROVED** [review]
Just one question about the button variant...

File: src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx:40]: > Just wanted to confirm that this is the correct variant, compared to the one in [the Figma screen cap on the card](https://www.notion.so/artsy/UX-mWeb-Empty-state-is-different-from-Figma-design-c1dcb05813fc45d9aec709b57c2fa68c)?

yep, it's correct button variant (we discussed it today with Ryan)

Comment (dimatretyak) [reply src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx:40]: > I've just realised the primary grey button has been removed from the design system

Received message from Ryan today

**Pull request merged**